src/commands/synthetics/README.md

```diff
@@ -355,29 +355,31 @@ Your test files must be named with a `.synthetics.json` suffix.
+| `startUrlSubstitutionRegex`        | String           | The regex to modify the starting URL of the test (for browser and HTTP tests only), whether it was given by the original test or the configuration override `startUrl`. <br><br>If the URL contains variables, this regex applies after the interpolation of the variables. There are two possible formats: <br>- `your_regex\|your_substitution`: The pipe-based syntax, to avoid any conflicts with `/` characters in URLs. For example, `https://example.com(.*)\|http://subdomain.example.com$1` to transform `https://example.com/test` to `http://subdomain.example.com/test`. <br>- `s/your_regex/your_substitution/modifiers`: The slash syntax, which supports modifiers. For example, `s/(https://www.)(.*)/$1extra-$2/` to transform `https://www.example.com` into `https://www.extra-example.com`. | `DATADOG_SYNTHETICS_OVERRIDE_START_URL_SUBSTITUTION_REGEX` | `--override startUrlSubstitutionRegex="s/(https://www.)(.*)/$1extra-$2/"`  |
+| `testTimeout`                      | Number           | The maximum duration of a browser test in seconds.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | `DATADOG_SYNTHETICS_OVERRIDE_TEST_TIMEOUT` | `--override testTimeout=300`  |
+| `variables`                        | Object           | The variables to replace in the test. This object should contain key as the name of the variable to replace and values as the new value of the variable to replace.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | - | - |
```

Comment:
```suggestion
| Options | Type | Definition | Environment Variable | CLI Option |
|---|---|---|---|---|
| `allowInsecureCertificates` | Boolean | Disable certificate checks in Synthetic API and browser tests. | `DATADOG_SYNTHETICS_OVERRIDE_ALLOW_INSECURE_CERTIFICATES` | `--override allowInsecureCertificates=true` |
| `basicAuth` | Object | Credentials to provide if basic authentication is required.<br><br>- `username` (String): The username for basic authentication.<br>- `password` (String): The password for basic authentication. | - | - |
| `body` | String | Data to send in an API test. | `DATADOG_SYNTHETICS_OVERRIDE_BODY` | `--override body= "{\"fakeContent\":true}"` |
| `bodyType` | String | Content type for the data to send in an API test. | `DATADOG_SYNTHETICS_OVERRIDE_BODY_TYPE` | `--override bodyType="application/json"` |
| `cookies` | String or object | Use the provided string as a cookie header in an API or browser test (in addition or as a replacement).<br><br>- If this is a string, it is used to replace the original cookies.<br>- If this is an object, the format must be `{append?: boolean, value: string}`, and depending on the value of `append`, it is appended or replaces the original cookies. | - | - |
| `defaultStepTimeout` | Number | The maximum duration of steps in seconds for browser tests, which does not override individually set step timeouts. | `DATADOG_SYNTHETICS_OVERRIDE_DEFAULT_STEP_TIMEOUT` | `--override defaultStepTimeout=15` |
| `deviceIds` | Array | A list of devices to run the browser test on. | - | - |
| `executionRule` | String | The execution rule for the test defines the behavior of the CLI in case of a failing test.<br><br>- `blocking`: The CLI returns an error if the test fails.<br>- `non_blocking`: The CLI only prints a warning if the test fails.<br>- `skipped`: The test is not executed at all. | `DATADOG_SYNTHETICS_OVERRIDE_EXECUTION_RULE` | `--override followRedirects=skipped` |
| `followRedirects` | Boolean | Indicates whether or not to follow HTTP redirections in Synthetic API tests. | `DATADOG_SYNTHETICS_OVERRIDE_FOLLOW_REDIRECTS` | `--override followRedirects=true` |
| `headers` | Object | The headers to replace in the test. This object should contain keys as the name of the header to replace and values as the new value of the header to replace. | - | - |
| `locations` | Array | A list of locations to run the test from. | - | - |
| `mobileApplicationVersion` | String | Override the default mobile application version for a Synthetic mobile application test. The version must be uploaded and available within Datadog. | - | - |
| `mobileApplicationVersionFilePath` | String | Override the application version for a Synthetic mobile application test. | - | - |
| `pollingTimeout` | Integer | The maximum duration in milliseconds of a test. If the execution exceeds this value, it is considered failed. The maximum duration is 7,200,000 milliseconds, or 2 hours. | - | - |
| `retry` | Object | The retry policy for the test.<br><br>- `count` (Integer): The number of attempts to perform in case of test failure.<br>- `interval` (Integer): The interval between attempts in milliseconds. | - `DATADOG_SYNTHETICS_OVERRIDE_RETRY_COUNT`<br>- `DATADOG_SYNTHETICS_OVERRIDE_RETRY_INTERVAL` | `--override retry.count=2 --override retry.interval=300` |
| `startUrl` | String | The new start URL to provide to the test. Variables specified in brackets (for example, `{{ EXAMPLE }}`) found in environment variables are replaced. | `DATADOG_SYNTHETICS_OVERRIDE_START_URL` | `--override startUrl="{{ EXAMPLE }}"` |
| `startUrlSubstitutionRegex` | String | The regex to modify the starting URL of the test (for browser and HTTP tests only), whether it was given by the original test or the configuration override `startUrl`. <br><br>If the URL contains variables, this regex applies after the interpolation of the variables. <br><br>There are two possible formats: <br><br>- `your_regex\|your_substitution`: The pipe-based syntax, to avoid any conflicts with `/` characters in URLs. For example, `https://example.com(.*)\|http://subdomain.example.com$1` to transform `https://example.com/test` to `http://subdomain.example.com/test`. <br>- `s/your_regex/your_substitution/modifiers`: The slash syntax, which supports modifiers. For example, `s/(https://www.)(.*)/$1extra-$2/` to transform `https://www.example.com` into `https://www.extra-example.com`. | `DATADOG_SYNTHETICS_OVERRIDE_START_URL_SUBSTITUTION_REGEX` | `--override startUrlSubstitutionRegex="s/(https://www.)(.*)/$1extra-$2/"` |
| `testTimeout` | Number | The maximum duration of a browser test in seconds. | `DATADOG_SYNTHETICS_OVERRIDE_TEST_TIMEOUT` | `--override testTimeout=300` |
| `variables` | Object | The variables to replace in the test. This object should contain key as the name of the variable to replace and values as the new value of the variable to replace. | - | - |
```